src/plugins/initiatives/initiativeEntry.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import {type TimestampMs} from "../../util/timestamp";
+import {type NodeAddressT} from "../../core/graph";
+import {type InitiativeWeight} from "./initiative";
+import {type NodeEntryJson} from "./nodeEntry";
+
+/**
+ * Stored inside a initiatives.json db file with other initiatives.
+ * presently the `id` prop is used as the key inside the db file
+ */
+export type InitiativeEntry = {|
+  +id: string, // GUID
+  +title: string,
+  +timestampMs: TimestampMs,
+  +weight: InitiativeWeight,
+  +completed: boolean,
+  // should be user nodes
+  +champions: $ReadOnlyArray<NodeAddressT>,
+  // other initiative nodes
+  +dependencies: $ReadOnlyArray<NodeAddressT>,
+  // an activity node. Will be more difficult to specify this
+  +references: $ReadOnlyArray<NodeAddressT>,
+  +contributions: $ReadOnlyArray<NodeEntryJson>,
```

Comment:
The NodeEntryJson type uses url references, not node address references. Let's please add a new way of representing inline contributions, which does node address references.

Reply:
Also, let's have the new type be precise / exact (i.e. it will have every field) rather than making all the fields optional -- for the same reason as here, if we are not trying to ensure that the type is easy for manual human editing, then it is cleaner to just always have the fields.